draft-ietf-quic-multipath.md

```diff
@@ -381,16 +378,20 @@ with the same, unused Path ID. When the peer receives the PATH_CHALLENGE,
 it MUST pick a Connection ID with the same Path ID for sending the PATH_RESPONSE.
 
 When the multipath extension is negotiated, a client that wants to use an
-additional path MUST first initiate the Address Validation procedure
+additional path SHOULD first initiate the Address Validation procedure
 with PATH_CHALLENGE and PATH_RESPONSE frames as described in
-{{Section 8.2 of QUIC-TRANSPORT}}, unless it has previously validated
+{{Section 8.2 of QUIC-TRANSPORT}}. It MAY skip the validation and
+simply send packets on the paths if it has previously validated
 that address.
 
 After receiving packets from the
 client on a new path, if the server decides to use the new path,
-the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
+the server SHOULD perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
```

Comment:
Similarly we should keep the MUST here or not use normative language because this is also covered by the MUST above.

Reply:
I am concerned that receivers will treat the "PATH CHALLENGE" as a trigger to create a path context, and will drop packets that do not contain a path challenge. That can easily lead to interop issues.

Reply:
The important part here is the "if a server decides to use the new path". However, we can also rephrase this and say more explicitly "before sending on a new path" or something.

But I actually thought the whole point of this PR was to explicitly clarify that even packets without a path challenge can create a new path as the other end is obligated to perform address validation but a) it can and shouldn't be enforced and b) it can be done by other means than sending a path challenge. Therefore to re-add this text to say valid packets have to be processed is probably good to ensure this is crystal clear.